src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -0,0 +1,167 @@
+                        iconHeight: 40,
+                    };
+                    break;
+                }
+                case CONST.POLICY.CONNECTIONS.NAME.XERO: {
+                    text = translate('workspace.accounting.xero');
+                    onboardingIcon = {
+                        icon: Expensicons.XeroCircle,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                    };
+                    break;
+                }
+                case CONST.POLICY.CONNECTIONS.NAME.NETSUITE: {
+                    text = translate('workspace.accounting.netsuite');
+                    onboardingIcon = {
+                        icon: Expensicons.NetSuiteSquare,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                        iconStyles: [StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR)],
+                    };
+                    break;
+                }
+                default: {
+                    text = translate('workspace.accounting.intacct');
+                    onboardingIcon = {
+                        icon: Expensicons.IntacctSquare,
+                        iconWidth: 40,
+                        iconHeight: 40,
+                        iconStyles: [StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR)],
+                    };
+                    break;
+                }
+            }
+            return {
+                keyForList: connectionName,
+                text,
+                onboardingIcon,
+                isSelected: userReportedIntegration === connectionName,
+            };
+        });
+        const noneAccountingOption: OnboardingListItemData = {
+            text: translate('onboarding.accounting.noneOfAbove'),
+            keyForList: null,
+            onboardingIcon: {
+                icon: Expensicons.Clear,
+                iconFill: theme.success,
+            },
+            isSelected: userReportedIntegration === null,
+        };
+        return [...policyAccountingOptions, noneAccountingOption];
+    }, [StyleUtils, theme.success, translate, userReportedIntegration]);
+
+    const footerContent = (
+        <>
+            {!!error && (
+                <FormHelpMessage
+                    style={[styles.ph1, styles.mb2]}
+                    isError
+                    message={error}
+                />
+            )}
+            <Button
+                success
+                large
+                text={translate('common.confirm')}
+                // eslint-disable-next-line rulesdir/prefer-early-return
+                onPress={() => {
+                    if (!userReportedIntegration) {
+                        setError(translate('onboarding.purpose.errorSelection'));
+                        // eslint-disable-next-line no-useless-return
+                        return;
+                    }
```

Comment:
Check against `undefined` instead of falsy because `null` is a valid value (when selecting none of the above)